website/docs/Software/CompatibleThirdPartySoftware/04-LSL.md

```diff
@@ -2,79 +2,59 @@
 
 ## Setup
 
-1.  **Download or clone the [OpenBCI_LSL](https://github.com/openbci-archive/OpenBCI_LSL) repo from Github.**
+1.  **Download or clone the [OpenBCI_GUI](https://github.com/OpenBCI/OpenBCI_GUI) repo from Github.**
 2.  **Download and install [Python](https://www.python.org/downloads/) (either version 2 or 3).**
     Python might already be installed on your computer. Type python --version to check if you have Python version 2 or 3 installed.
 3.  **Install Python requirements**
     To use this program, you need the following Python packages installed:
 
--   pylsl (version 1.10.5 or greater)
--   pyserial (version 3.1.1 or greater)
--   numpy (version 1.11.1 or greater)
--   pyqtgraph (version 0.9.10 or greater) (optional: needded for GUI functionality only)
--   scipy (version 0.17.1 or greater) (optional: needed for GUI functionality only)
+- pylsl (version 1.10.5 or greater)
+- pyserial (version 3.1.1 or greater)
+- numpy (version 1.11.1 or greater)
+- brainflow (version 5.10.0 or greater)
 
-To automatically install using pip, navigate to the "OpenBCI_LSL" folder on your command line and terminal, and type:
+To automatically install using pip, navigate to the "Networking-Test-Kit/LSL/" folder on your command line and terminal, and type:
 
 ```
 
-pip install -r requirements.txt
+pip install --upgrade numpy brainflow pylsl
 
 ```
 
-Note: pip may have issues install numpy and scipy for some users. Install these manually if you have issues.
-
-Note: If you get the message "pip: command not found", you need to install pip: sudo easy_install pip. Then retry the command above.
+:::note
+`pip` may have issues installing `numpy` and `scipy` for some users. Install these manually if you have issues.
 
-To use the GUI features of this application, you must separately install PyQt4 using these instructions: Install PyQt4. The command line version of this program is still functional if PyQt4 is not installed.
+If you get the message `pip: command not found`, you need to install pip using `sudo easy_install pip`. Then, retry the command above.
+:::
 
 ## Usage
 
 ### Simple Stream
 
-First, make sure your dongle is plugged in and board is powered on. Then go the the "OpenBCI_LSL" folder and type the following command:
-
-```
+First, make sure your dongle is plugged in and board is powered on. Then, type the following command:
 
-python openbci_lsl.py --stream
+Windows:
 
 ```
-
-After a few moments, you should see this output:
-
-<img src="https://raw.githubusercontent.com/gabrielibagon/OpenBCI_LSL/master/images/CLI.jpg?raw=true" width="80%" />
-
-If an error is raised about not being able to find your the board or serial port, you can override the automatic board detection by specifying the serial port in a command line argument before "--stream". The format is as follows:
-
+python3 Networking-Test-Kit/LSL/brainflow_lsl.py --board-id 2 --serial-port COM3 --name test --data-type EXG --channel-names 1,2,3,4,5,6,7,8 --uid brainflow
```

Comment:
Do not mention this complicated BrainFlow + LSL example that bypasses the GUI. Just mention lslStreamTest.py